source/partials/primary-domain.md

```diff
@@ -1,5 +1,11 @@
 ### Set a Primary Domain via the Dashboard
 
+<Alert type="danger" title="Warning">
+
+With a Primary Domain set at the platform level, all other domains will be pointed to your Primary domain _at the root level_. If you want to redirect secondary domains to specific pages on your site (for example, `olddomain.com` to `newdomain.com/old-landing-page`), do not set a Primary Domain. Instead use [PHP redirects](/redirects/#redirect-with-php).
```

Comment:
Suggest applying a similar warning to the new advanced docs page at the top, which says if they're going to do advanced redirect scenarios the primary domain feature needs to be set to none.

Reply:
Added in https://github.com/pantheon-systems/documentation/pull/5516/commits/a7e4c20988a819b75567c926d7678d71063e686a